package.json

```diff
@@ -31,5 +33,13 @@
   "bugs": {
     "url": "https://github.com/PrismarineJS/prismarine-item/issues"
   },
-  "homepage": "https://github.com/PrismarineJS/prismarine-item#readme"
+  "homepage": "https://prismarinejs.org/prismarine-item",
+  "dependencies": {
+    "prismarine-jsdoc-theme": "^2.3.12"
+  },
+  "standard": {
+    "ignore": [
+     "docs/"
+    ]
+  }
```

Comment:
I don't feel like we should ignore docs directory. Your theme uses prettier, which we do not support. I think that docs should remain consistent with the source code if it comes to code style. Using different code styles in docs will result in many PRs that aren't linted correctly and overall bad habits of possible future contributors.